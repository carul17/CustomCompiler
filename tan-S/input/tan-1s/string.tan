main {
	const s := "ghjdfh \n nbsjh";
	var t := "sfd098#varfgs";
	const c := 's';
	
	print s \n;
	print t \n;
	print c \n;
}