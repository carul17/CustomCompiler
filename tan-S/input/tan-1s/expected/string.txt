ghjdfh \n nbsjh
sfd098#varfgs
s
